app/views/layouts/hyrax.html.erb

```diff
@@ -1,6 +1,6 @@
 <!-- add body classes to make styling easier -->
 <!DOCTYPE html>
-<html lang="<%= I18n.locale.to_s %>" prefix="og:http://ogp.me/ns#">
+<html lang="<%= I18n.locale.to_s || :en %>" prefix="og:http://ogp.me/ns#">
```

Comment:
if this does work, is there a way to set a default locale, instead of having a fallback?

Reply:
the default locale is set in an initializer. It's not being recognized on staging